android/cli/hooks/aar-transform.js

```diff
@@ -41,9 +41,10 @@ const LIBRARY_ORIGIN_PORJECT = 'Project';
 
 exports.cliVersion = '>=3.2';
 
-exports.init = function (logger, config, cli) {
+exports.init = function (logger, config, cli, appc) {
```

Comment:
The varying argument order for the same objects bugs me:
`(logger, config, cli, appc)` vs
`(cli, builder, appc, logger)` vs 
`(builder, callback)` vs
`(builder, logger, callback)`

I prefer to try and retain argument order relatively if possible, so I guess in these cases it would be:
`(logger, config, cli, appc, callback)` - where `builder` falls, I'm not sure beyond having it before `callback`